plugins/aws/fix_plugin_aws/resource/backup.py

```diff
@@ -558,6 +656,43 @@ def called_collect_apis(cls) -> List[AwsApiSpec]:
             AwsApiSpec(service_name, "list-tags"),
         ]
 
+    @classmethod
+    def called_mutator_apis(cls) -> List[AwsApiSpec]:
+        return [
+            AwsApiSpec(service_name, "tag-resource"),
+            AwsApiSpec(service_name, "untag-resource"),
+            AwsApiSpec(service_name, "delete-report-plan"),
+        ]
+
+    def update_resource_tag(self, client: AwsClient, key: str, value: str) -> bool:
+        client.call(
+            aws_service=self.api_spec.service,
+            action="tag-resource",
+            result_name=None,
+            ResourceArn=self.arn,
+            Tags={key: value},
+        )
+        return True
+
+    def delete_resource_tag(self, client: AwsClient, key: str) -> bool:
+        client.call(
+            aws_service=self.api_spec.service,
+            action="untag-resource",
+            result_name=None,
+            ResourceArn=self.arn,
+            TagKeyList=[key],
+        )
+        return True
```

Comment:
can we extract a separate base class? Seems to be repeated.
See CloudwatchTaggable as example